modules/local/align/bwamem2.nf

```diff
@@ -0,0 +1,55 @@
+process ALIGN_BWAMEM2 {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mulled-v2-e5d375990341c5aef3c9aff74f96f66f65375ef6:2d15960ccea84e249a150b7f5d4db3a42fc2d6c3-0' :
+        'biocontainers/mulled-v2-e5d375990341c5aef3c9aff74f96f66f65375ef6:2d15960ccea84e249a150b7f5d4db3a42fc2d6c3-0' }"
+
+    input:
+    tuple val(meta), path(reads)
+    tuple val(meta2), path(index)
+
+    output:
+    tuple val(meta), path("*_sorted.bam"), path("*_sorted.bam.bai"), emit: bam
+    path  "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = '-M'
+    def args2 = '-F256 -F4 -uS'
```

Comment:
since we are hard coding these args and not sourcing them from a config file, why not hard code them straight in the tool call below?

Reply:
Agree!